publisher-sdk/src/test/java/com/criteo/publisher/headerbidding/HeaderBiddingTest.kt

```diff
@@ -64,7 +66,7 @@ class HeaderBiddingTest {
 
     val headerBidding = HeaderBidding(bidManager, listOf(handler), integrationRegistry)
 
-    headerBidding.enrichBid(obj, adUnit)
+    headerBidding.enrichBid(obj, adUnit, mock())
```

Comment:
That's not what happens currently though? Currently, if there is a nofill, then the `setBidForAdUnit` returns and doesn't do anything, which makes sense because in this case, we don't participate to the server side auction.

I can add a second callback method to the `BidListener` called: onNoBid(), but we will need to think about how it will be used by the publishers. Let's talk about that tomorrow.

Reply:
Yep indeed. Will correct.